### PR TITLE
update RBAC documentation after introducing new roles

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 1]
```diff
@@ -19,14 +19,20 @@ Mender supports four different types of Roles:
 
 * **Admin:** Full access
 
-* **CI:** Intended for *Continuous Integration* systems. It can only manage
-Mender Artifacts, such as upload and delete Artifacts.
-
-* **Read only:** Allows the user to see the status of devices and deployments,
+* **Read Access:** Allows the user to see the status of devices and deployments,
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:22]: From what I understood Fabio, this should be in the description to:

> Unless the user is assigned Admin, this role is needed to allow access to the UI.

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 2]
```diff
@@ -19,14 +19,20 @@ Mender supports four different types of Roles:
...
+* **Release Manager:** Intended for *Continuous Integration* systems. It can only manage
+Mender Artifacts, such as upload and delete Artifacts.
+
+* **Deployments Manager:** Intended for users responsible for managing deployments.
+With this role users can create and abort deployments.
+
+* **Troubleshooting:** User with this role assigned has access
+to the troubleshooting features such as Remote Terminal, File Transfer, Port Forwarding.
+
 * **Device groups:** Grants access to a *customizable* set of Device groups in
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:36]: For the device groups I think it's important to add how to actually get a result where "a user can achieve full control but only over a single group".  That part isn't evident.

Would expand with:
> To have a user who has full control over a particular group and nothing else be sure to give to give them the roles: Deployments Manager, Read Access and the chosen Device group role.

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 1 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: updated

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 2 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md:38]: updated

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 3]
```diff
@@ -19,21 +19,31 @@ Mender supports four different types of Roles:
...
-* **Read only:** Allows the user to see the status of devices and deployments,
+* **Read Access:** Unless the user is assigned Admin, this role is needed
+to allow access to the UI. The role allows the user to see the status of devices and deployments,
 but not make any modifications. This role is well suited for limited technical
 support users, or team leads who need an overview of deployment status or
 individual devices, but are not involved in day-to-day deployment management.
 
+
+* **Release Manager:** Intended for *Continuous Integration* systems. It can only manage
+Mender Artifacts, such as upload and delete Artifacts.
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:30]:
```suggestion
Mender Artifacts, such as upload and delete Artifacts. On its own doesn't allow UI access.
```

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 4]
```diff
@@ -19,21 +19,31 @@ Mender supports four different types of Roles:
...
+
+* **Deployments Manager:** Intended for users responsible for managing deployments.
+With this role users can create and abort deployments.
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:33]:
```suggestion
With this role users can create and abort deployments.  On its own doesn't allow UI access.
```

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 5]
```diff
@@ -19,21 +19,31 @@ Mender supports four different types of Roles:
...
+
+* **Troubleshooting:** User with this role assigned has access
+to the troubleshooting features such as Remote Terminal, File Transfer, Port Forwarding.
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:36]:
```suggestion
to the troubleshooting features such as Remote Terminal, File Transfer, Port Forwarding.  On its own doesn't allow UI access.
```

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 3 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: done

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 4 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: done

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 5 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md:36]: done

**Comment by TheMeaningfulEngineer** [comment]
I'm putting you through a hard time @kjaskiewiczz  I am sorry :(
I've just tested this and `On its own doesn't allow UI access.` isn't correct.
You can log in now, previously this wasn't possible with just the Device groups. 

I'll make suggestions you can just apply with a click :)

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 6]
```diff
@@ -19,21 +19,32 @@ Mender supports four different types of Roles:
...
-* **Read only:** Allows the user to see the status of devices and deployments,
+* **Read Access:** Unless the user is assigned Admin, this role is needed
+to allow access to the UI. The role allows the user to see the status of devices and deployments,
 but not make any modifications. This role is well suited for limited technical
 support users, or team leads who need an overview of deployment status or
 individual devices, but are not involved in day-to-day deployment management.
 
+
+* **Release Manager:** Intended for *Continuous Integration* systems. It can only manage
+Mender Artifacts, such as upload and delete Artifacts. On its own doesn't allow UI access.
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:30]:
```suggestion
Mender Artifacts, such as upload and delete Artifacts.
```

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 7]
```diff
@@ -19,21 +19,32 @@ Mender supports four different types of Roles:
...
+
+* **Deployments Manager:** Intended for users responsible for managing deployments.
+With this role users can create and abort deployments. On its own doesn't allow UI access.
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:33]:
```suggestion
With this role users can create and abort deployments. On it's own this role won't make the devices visible in the UI, you must pair it with Read Access for that```

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 8]
```diff
@@ -19,21 +19,32 @@ Mender supports four different types of Roles:
...
+
+* **Troubleshooting:** User with this role assigned has access
+to the troubleshooting features such as Remote Terminal, File Transfer, Port Forwarding.
+On its own doesn't allow UI access.
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:37]:
```suggestion
On it's own this role won't make the devices visible in the UI, you must pair it with Read Access for that.
```

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 9]
```diff
@@ -19,21 +19,32 @@ Mender supports four different types of Roles:
 
 * **Admin:** Full access
 
-* **CI:** Intended for *Continuous Integration* systems. It can only manage
-Mender Artifacts, such as upload and delete Artifacts.
-
-* **Read only:** Allows the user to see the status of devices and deployments,
+* **Read Access:** Unless the user is assigned Admin, this role is needed
+to allow access to the UI. The role allows the user to see the status of devices and deployments,
```
Comment (TheMeaningfulEngineer) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:23]:
```suggestion
* **Read Access:** The role allows the user to see the status of devices and deployments,
```

**Comment by TheMeaningfulEngineer** [comment]
Ok, I think this is it.
I've tested it and that's how they behave. :)

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 6 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: done

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 7 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: done

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 8 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: done

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 9 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: done

**Review by TheMeaningfulEngineer — APPROVED** [review]
Great, thanks

**Pull request merged**